OpenDocumentReader/DocumentViewController.swift

```diff
@@ -107,6 +116,40 @@ class DocumentViewController: UIViewController, DocumentDelegate {
         return isFullscreen
     }
     
+    func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
+        searchBar.text = ""
+        searchBar.isHidden = true
```

Comment:
move code for hiding / showing searchbar into one method to avoid code duplication